azure/functions/decorators/eventhub.py

```diff
@@ -0,0 +1,43 @@
+#  Copyright (c) Microsoft Corporation. All rights reserved.
+#  Licensed under the MIT License.
+from typing import Optional
+
+from azure.functions.decorators import Cardinality
```

Comment:
You should import this from the module which contains its definition directly - to prevent any possibility of a cyclic dependency - as the `__init__.py` could be importing something from this file (I know it doesn't in this file, but for good practice).